data/lib/core/town.lua

```diff
@@ -0,0 +1,24 @@
+Town = {}
```

Comment:
This file is never read, it must be added to the core.lua file

Reply:
It is also necessary to add the `__index` metamethod that refers to the same table
```lua
Town.__index = Town
```

Reply:
Fixed